src/QsCompiler/Transformations/IntrinsicResolutionTransformation.cs

```diff
@@ -9,9 +9,9 @@
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
 
 
-namespace Microsoft.Quantum.QsCompiler.Transformations.IntrinsicResolutionTransformation
+namespace Microsoft.Quantum.QsCompiler.Transformations.IntrinsicResolution
 {
-    public class IntrinsicResolutionTransformation
+    public class ReplaceWithTargetIntrinsics
```

Comment:
It seems like this class is not actually a transformation. I'm guessing it's a utility class? Could it be made static?